driver/k8s/logicalvolume_service.go

```diff
@@ -116,10 +116,6 @@ func (s *LogicalVolumeService) CreateVolume(ctx context.Context, node, dc, name,
 	// if the create volume request has no source, proceed with regular lv creation.
 	if sourceName == "" {
 		lv = &topolvmv1.LogicalVolume{
-			TypeMeta: metav1.TypeMeta{
```

Comment:
Why did you delete these lines instead of setting `APIVersion` to `topolvm.io/v1`?

Reply:
@bells17 
Could you give me a comment for this?

Reply:
Sorry for the late reply.
When using a typed object like a `LogicalVolume`, a controller-runtime client uses a typedClient in internal.
typedClient select a REST client using `reflect.ValueOf(obj).Elem().Type()` instead of TypeMeta GVK.
So no need set a TypeMeta field.

ref:
- https://github.com/kubernetes-sigs/controller-runtime/blob/v0.12.1/pkg/client/typed_client.go#L38
- https://github.com/kubernetes/apimachinery/blob/v0.24.2/pkg/runtime/scheme.go#L257-L263